src/WCSimPMTObject.cc

```diff
@@ -1634,3 +1634,331 @@ G4float BoxandLine12inchHQE::GetDarkRateConversionFactor(){
+      0.999313, 0.999321, 0.999331, 0.999347, 0.999356,
+      0.999369, 0.999381, 0.999394, 0.999402, 0.999415,
+      0.999427, 0.999433, 0.999446, 0.999458, 0.999472,
+      0.999484, 0.999499, 0.999513, 0.999522, 0.999532,
+      0.999540, 0.999550, 0.999559, 0.999567, 0.999574,
+      0.999588, 0.999599, 0.999613, 0.999618, 0.999627,
+      // 8
+      0.999635, 0.999639, 0.999652, 0.999662, 0.999667,
+      0.999671, 0.999678, 0.999682, 0.999688, 0.999693,
+      0.999698, 0.999701, 0.999706, 0.999711, 0.999718,
+      0.999722, 0.999727, 0.999732, 0.999737, 0.999740,
+      0.999746, 0.999750, 0.999754, 0.999763, 0.999766,
+      0.999769, 0.999774, 0.999780, 0.999784, 0.999788,
+      0.999796, 0.999803, 0.999807, 0.999809, 0.999815,
+      0.999820, 0.999827, 0.999830, 0.999833, 0.999833,
+      0.999836, 0.999839, 0.999842, 0.999845, 0.999850,
+      0.999853, 0.999857, 0.999860, 0.999865, 0.999870,
+      // 9
+      0.999873, 0.999877, 0.999880, 0.999882, 0.999883,
+      0.999886, 0.999888, 0.999889, 0.999895, 0.999896,
+      0.999897, 0.999901, 0.999902, 0.999905, 0.999907,
+      0.999907, 0.999909, 0.999911, 0.999911, 0.999912,
+      0.999913, 0.999914, 0.999917, 0.999919, 0.999921,
+      0.999923, 0.999927, 0.999929, 0.999931, 0.999933,
+      0.999936, 0.999942, 0.999942, 0.999944, 0.999947,
+      0.999947, 0.999948, 0.999949, 0.999952, 0.999955,
+      0.999957, 0.999957, 0.999961, 0.999962, 0.999963,
+      0.999963, 0.999963, 0.999964, 0.999965, 0.999965,
+      // 10
+      0.999965, 0.999965, 0.999966, 0.999968, 0.999969,
+      0.999971, 0.999972, 0.999972, 0.999973, 0.999975,
+      0.999975, 0.999975, 0.999975, 0.999975, 0.999975,
+      0.999975, 0.999979, 0.999979, 0.999980, 0.999982,
+      0.999983, 0.999985, 0.999986, 0.999987, 0.999987,
+      0.999988, 0.999989, 0.999989, 0.999989, 0.999989,
+      0.999990, 0.999990, 0.999992, 0.999993, 0.999994,
+      0.999994, 0.999994, 0.999994, 0.999994, 0.999995,
+      0.999995, 0.999995, 0.999996, 0.999996, 0.999996,
+      0.999996, 0.999998, 0.999999, 1.000000, 1.000000,
+      // Dummy element for noticing if the loop reached the end of the array
+      0.0
+  };
+  return qpe0;
+}
+
+//Currenly the PMT QE info is the same as 20 inch.
+G4float* PMT5inch::GetQE(){
+  static G4float QE[20] = { 0.00, .0139, .0854, .169, .203, .206, .211, .202,.188, .167, .140, .116, .0806, .0432, .0265, .0146, .00756, .00508, .00158, 0.00};
+  return QE;
+}
+G4float* PMT5inch::GetQEWavelength(){static G4float wavelength[20] = { 280., 300., 320., 340., 360., 380., 400., 420., 440., 460., 480., 500., 520., 540., 560., 580., 600., 620., 640., 660.};
+  return wavelength;}
+
+G4float  PMT5inch::GetmaxQE(){
+  const G4float maxQE = 0.211;
+  return maxQE;
+}
+
+// Should be actual PMT Dark Rate, not effective dark rate in detector including other LE noise
+G4float PMT5inch::GetDarkRate(){
+  const G4float rate = 400*CLHEP::hertz;   //SKI value set in SKDETSim.
```

Comment:
Is this a reasonable dark rate to use for these tubes?

Reply:
I used the measurements we made at QM from the ETEnterprise PMT's prototype we plan to use on the OD. Perhaps I can rename them to include the fact they came from ET ?